rust/noosphere/src/ffi/fs.rs

```diff
@@ -241,3 +242,36 @@ pub fn ns_sphere_file_header_values_read(
         .into_boxed_slice()
         .into()
 }
+
+#[ffi_export]
+/// Get the first header value for a file that corresponds to the given name, if any
```

Comment:
nit: *Get the first header value that corresponds to the given name for a file.*